fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -93,7 +93,7 @@ class ApplyMetadataMutationsImpl {
 	    storageCache(&proxyCommitData_.storageCache), tag_popped(&proxyCommitData_.tag_popped),
 	    tssMapping(&proxyCommitData_.tssMapping), tLogGroupCollection(proxyCommitData_.tLogGroupCollection),
 	    initialCommit(initialCommit_), tagToServer(&proxyCommitData_.tagToServer),
-	    ssToStorageTeam(&proxyCommitData_.ssToStorageTeam) {
+	    ssToStorageTeam(&proxyCommitData_.ssToStorageTeam) /* unused , changedTeams(&proxyCommitData_.changedTeams) */ {
```

Comment:
`changedTeams` has been removed from your PR: https://github.com/apple/foundationdb/pull/6657

Just remove this commented code and below related to `changedTeams`.